best-practices.md

```diff
@@ -0,0 +1,55 @@
+# Best practices when creating a workflow
+A workflow should be developed in accordance with the guidelines outlined in the  [serverless workflow definitions][1] documentation.
+
+This document provides a summary of several additional rules and recommendations to ensure smooth integration with other applications, such as the [Backstage Orchestrator][2].
```

Comment:
>  such as the [Backstage Orchestrator][2].

Is it correct to link workflow-result-schema.json ? From the sentence I expect a link to the orchestrator operator or plugin: can you clarify?
In addition, not sure if `Backstage Orchestrator` is already used: better to use a consistent naming, can you double check?

Reply:
Good catch, it was a mistake.
We are newly referencing the 1.2 helm chart.

Reply:
@masayag WDYT about this line?

Reply:
looks good now.